common-modules/src/main/java/com/softserve/if072/common/model/id/ShoppingListId.java

```diff
@@ -0,0 +1,33 @@
+package com.softserve.if072.common.model.id;
+
+import com.softserve.if072.common.model.Product;
+import com.softserve.if072.common.model.User;
+
+import java.io.Serializable;
+
+/**
+ * Created by Nazar Vynnyk
+ */
+public class ShoppingListId implements Serializable {
+
+    private User user;
+    private Product product;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)